src/css/App.css

```diff
@@ -10,12 +10,22 @@ Orange #fdba35
 Blue Teal #3ebab4
 Medium Gray #5e666c
 
+NEW PALLETTE: 
+Dark Gold: #EEAD0E
+Light Gold: #F8DE9E
+Dark Gray: #bbbbbb
+Light Gray: #cfcfcf
+Brown: #7d6c54
+Black: #1d2023
+White: #ffffff
+
 *********************/
```

Comment:
I think this would be a good use case for SASS.  These colors can be assigned to variables and then referenced throughout the css files.

Reply:
Oh, awesome, we can definitely go back and refactor with SASS.